src/debug.ts

```diff
@@ -91,8 +91,8 @@ export class DebugSessionsView implements TreeDataProvider<string>, Disposable {
         return undefined;
     }
 
-    refresh(): void {
-        this.onDidChangeTreeDataEmitter.fire();
+    refresh(element?: string): void {
+        this.onDidChangeTreeDataEmitter.fire(element);
```

Comment:
not needed, it never updates specific element, because elements in the debug view have no children. It always updates whole view.